authors/apps/authentication/serializers.py

```diff
@@ -25,11 +28,78 @@ class Meta:
         # or response, including fields specified explicitly above.
         fields = ['email', 'username', 'password']
 
+    def validate_password(self, password):
+        """ This function validates the password input by a new user signing up
+
+            It ensures that the password length is longer than 8 characters to be considered valid.
+            The password should also contain at least a letter and number. 
+
+        Args: 
+            param1 (str): password
```

Comment:
@d-rita please rename `param1` e.g 
```suggestion
            password (str):  This is password string received from user.
```